asserts/device_asserts.go

```diff
@@ -136,8 +128,8 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 	// ignore extra headers and non-empty body for future compatibility
 	return &Model{
 		assertionBase: assert,
-		allowedModes:  allowedModes,
-		requiredSnaps: requiredSnaps,
+		allowedModes:  nil, // XXX: empty for now
```

Comment:
if these are going to be assumed empty for now wouldn't it behoove us to warn if it's set to something non-empty?